### PR TITLE
gantt like doc

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**

### CODE REVIEW

File: docs/manuals/gui/viselements/charts/gantt.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+## Gantt charts
+
+Gantt charts are not supported by Plotly.
```
Comment (FabienLelaquais) [inline docs/manuals/gui/viselements/charts/gantt.md]: ->
"Although real Gantt charts are not supported by Plotly, there are ways to display horizontal bar charts with different starting points that can look like a Gantt chart."

File: docs/manuals/gui/viselements/charts/gantt.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+But Plotly provides a way of displaying horizontal bar charts with different starting points that can look like a Gantt chart.
+
+### Simple Gantt like chart
+
+Using a horizontal bar chart with different starting points will look like a Gantt chart.
+- _type_ property is set to bar.
```
Comment (FabienLelaquais) [inline docs/manuals/gui/viselements/charts/gantt.md]: set to "bar"

File: docs/manuals/gui/viselements/charts/gantt.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- _orientation_ property is set to h or horizontal.
```
Comment (FabienLelaquais) [inline docs/manuals/gui/viselements/charts/gantt.md]: "h" or "horizontal"

File: docs/manuals/gui/viselements/charts/gantt.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Different resources are displayed by different traces in the same chart.
+
+Here is an example where we represent two resources' tasks, based on the same
+_x_ timeline axis:
+
+- The tasks start dates are represented by the _base_ values on the _x_ axis (type is date or datetime ).
+- The tasks duration are represented by the _x_ values on the _x_ axis (type is date or datetime).
```
Comment (FabienLelaquais) [inline docs/manuals/gui/viselements/charts/gantt.md]: durations

File: docs/manuals/gui/viselements/charts/gantt.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Different resources are displayed by different traces in the same chart.
+
+Here is an example where we represent two resources' tasks, based on the same
+_x_ timeline axis:
+
+- The tasks start dates are represented by the _base_ values on the _x_ axis (type is date or datetime ).
+- The tasks duration are represented by the _x_ values on the _x_ axis (type is date or datetime). 
+  The _x_ values are specified as a time duration ie a date where 0 is January 1st 1970 ie `date(1970, 1, 1)`. for example to specify a duration of 2 days, one would set the value to date(1970, 1, 3)
+- The resources names are reprensented by the _y_ values. To keep the tasks on the same horizontal line, the _y_ value should be the same.
```
Comment (FabienLelaquais) [inline docs/manuals/gui/viselements/charts/gantt.md]: represented

File: docs/manuals/gui/viselements/charts/gantt.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+- The tasks names can by specified by the _text_or _label_ property.
```
Comment (FabienLelaquais) [inline docs/manuals/gui/viselements/charts/gantt.md]: "by the _text_or _label_ property" -> "by the _text_ or the _label_ property."

**Review by FabienLelaquais — APPROVED** [review]
Minor, minor changes. Great job!

File: docs/manuals/gui/viselements/charts/gantt.md (hunk 1 above)
Comment (FredLL-Avaiga) [reply docs/manuals/gui/viselements/charts/gantt.md]: tx

**Pull request merged**